From 445d613f27d3cf28291919f29ccfbe4482082ae5 Mon Sep 17 00:00:00 2001
From: Kei Okada <kei.okada@gmail.com>
Date: Sat, 21 Feb 2015 10:39:09 +0900
Subject: [PATCH] do not need to create .catkin for build install

---
 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 73306a5..07401bb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -48,7 +48,8 @@ catkin_package(CFG_EXTRAS euslisp-extras.cmake)
 catkin_add_env_hooks(99.euslisp SHELLS sh DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/env-hooks)
 
 string(REGEX MATCH "catkin" need_catkin "$ENV{_}")
-if(need_catkin)
+get_filename_component(install_dir_name ${CMAKE_INSTALL_PREFIX} NAME)
+if(need_catkin AND NOT install_dir_name MATCHES "install") # check if  install_prefix end with /install, we can  remove this section of we remove all rosrun euslisp ... usage
   install(CODE "
 ## this is tricky force write catkin marker file
 set(_catkin_marker_file \"\${CMAKE_INSTALL_PREFIX}/.catkin\")
-- 
1.9.1

