From b577266dfaef74e1dbb9852899e7cea2158bffaf Mon Sep 17 00:00:00 2001
From: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
Date: Tue, 31 Aug 2021 20:52:31 +0900
Subject: [PATCH 2/4] speed up arm/target, disabe h7 in test-cad.l

---
 .travis.sh | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/.travis.sh b/.travis.sh
index 046620ad..4380a6ef 100755
--- a/.travis.sh
+++ b/.travis.sh
@@ -210,6 +210,15 @@ set +e
 if [[ "`uname -m`" == "aarch"* ]]; then
     sed -i 's@00000@0000@' $CI_SOURCE_PATH/test/object.l $CI_SOURCE_PATH/test/coords.l
 fi
+# arm target (ubuntu_arm64/trusty) takes too long time (>50min) for test
+# osrf/ubuntu_arm64:trusty takes >50 min, reduce loop count for irteus-demo.l
+if [[ "$DOCKER_IMAGE" == *"arm"* ]]; then
+    sed -i 's@00000@0@' $CI_SOURCE_PATH/test/object.l $CI_SOURCE_PATH/test/coords.l
+    sed -i 's@do-until-key-counter 10@do-until-key-counter 1@' irteus/test/irteus-demo.l;
+    sed -i 's/h7/ape/' irteus/test/test-cad.l
+    sed -i 's/(hanoi-program (length \*disks\*))/(subseq (hanoi-program (length \*disks\*)) 0 2)/' irteus/demo/hanoi-arm.l
+    sed -i 's/^\s*footstep-list/(subseq footstep-list 0 3)/' irteus/demo/walk-motion.l
+fi
 
     # run test in EusLisp/test
     for test_l in $CI_SOURCE_PATH/test/*.l; do
-- 
2.17.1

