From 7876d67812f3d838196ea268171376d3a6f58bfd Mon Sep 17 00:00:00 2001
From: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
Date: Tue, 31 Aug 2021 21:24:22 +0900
Subject: [PATCH 4/4] skip test-collision.l/test-cad.l/graph.l for armhf

---
 .travis.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.travis.sh b/.travis.sh
index 4380a6ef..2d2f2f6c 100755
--- a/.travis.sh
+++ b/.travis.sh
@@ -277,6 +277,8 @@ fi
         # skip collision test because bullet of 2.83 or later version is not released in trusty and jessie.
         # https://github.com/euslisp/jskeus/blob/6cb08aa6c66fa8759591de25b7da68baf76d5f09/irteus/Makefile#L37
         if [[ ( "$DOCKER_IMAGE" == *"trusty"* || "$DOCKER_IMAGE" == *"jessie"* ) && $test_l =~ test-collision.l ]]; then export CONTINUE=1; fi
+        # skip if test-cad.l/graph.l for arm
+        if [[ "$DOCKER_IMAGE" == *"arm"* && $test_l =~ test-collision.l|test-cad.l|graph.l ]]; then export CONTINUE=1; fi
 
         if [[ $CONTINUE == 0 ]]; then travis_time_end `expr 32 - $TMP_EXIT_STATUS`; else travis_time_end 33; fi
 
-- 
2.17.1

